src/Extension.ts

```diff
@@ -70,5 +71,28 @@ export class Extension {
 				}
 			)
 		);
+
+		this.dispose.track(
+			vscode.commands.registerCommand(
+				"hediet.vscode-drawio.newDiagram", async () => {
+					const targetUri = await vscode.window.showSaveDialog({
+						saveLabel: "Create",
+						filters: {
+							"Diagrams": ["drawio"]
+						}
+					});
+					if(!targetUri) { return; }
+					try {
+						await vscode.workspace.fs.writeFile(targetUri, new TextEncoder().encode(""));
+						await vscode.commands.executeCommand("vscode.openWith", targetUri, "hediet.vscode-drawio-text");
+					}
+					catch(e) {
+						await vscode.window.showErrorMessage(
+							`File "${targetUri.toString()}" cannot open!`
```

Comment:
Better message: 
`Cannot create or open file "${...}"!`
I think you should still do `console.error("Cannot create or open file", e)` before that.